recipe/meta.yaml

```diff
@@ -380,6 +399,29 @@ outputs:
         # python-version-specific library (default location in SP_DIR symlinks back to this)
         - test -f $PREFIX/lib/libtorch_python${SHLIB_EXT}     # [unix]
 
+        # the smoke test script takes a bunch of env variables, defined below
```

Comment:
Could we expand this comment a bit to describe the need for the env vars a bit more. There are quite a few

May also help to link out to some docs if there are good references available

Reply:
done